src/components/oppfoelgingsdialoger/OppfoelgingsPlanerOversikt.js

```diff
@@ -54,11 +54,11 @@ export class OppfoelgingsPlanerOversikt extends Component {
             inaktiveDialoger,
         } = this.props;
         aktiveDialoger.sort((a, b) => {
-            return new Date(a.godkjentPlan.deltMedNavTidspunkt) < new Date(b.godkjentPlan.deltMedNavTidspunkt) ? 1 : -1;
+            return new Date(b.godkjentPlan.deltMedNAVTidspunkt) - new Date(a.godkjentPlan.deltMedNAVTidspunkt);
         });
 
         inaktiveDialoger.sort((a, b) => {
-            return new Date(a.godkjentPlan.deltMedNavTidspunkt) < new Date(b.godkjentPlan.deltMedNavTidspunkt) ? 1 : -1;
+            return new Date(b.godkjentPlan.deltMedNAVTidspunkt) < new Date(a.godkjentPlan.deltMedNAVTidspunkt);
```

Comment:
Noen må si ifra når jeg gjør sånne teite ting som det her, det funker ikke når jeg bruker `<` i stedet for `-` 😬